{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/__init__.py

```diff
@@ -1,5 +1,8 @@
 """The main module for {{ cookiecutter.project_name }}."""
 
-from importlib.metadata import version
+from importlib.metadata import PackageNotFoundError, version
 
-__version__ = version(__name__)
+try:
+    __version__ = version("MUSE_OS")
+except PackageNotFoundError:
+    pass
```

Comment:
If we are ignoring the exception, then the following is more elegant and I think the recommended way of doing so:

```python 
from contextlib import suppress

with suppress(PackageNotFoundError):
    __version__ = version("MUSE_OS")
```

Reply:
And I bet we don't want to use `MUSE_OS`, right? ;)